Resources/doc/9e-integrating_several_parsers.md

```diff
@@ -0,0 +1,48 @@
+Step 9e: Integrating several parsers together
+=============================================
+
+Purpose
+--------
+Imagine that you need to integrate ExerciseHTMLPurifierBundle parser, bbcode parser, any other parser... it's better to have several parsers where each is responsible for it's action. That's why it's better to make chain of parsers, where order is important and parsers works one after other.
+
+Usage
+-----
+
+First, configure parser classes as services.
+
+```yaml
+    # See 9a-markup_htmlpurifier.md
+    markup.exercise_html_purifier:
+        class: FOS\CommentBundle\Markup\HtmlPurifier
+        arguments: [ @exercise_html_purifier.default ]
```

Comment:
this one should also be private (even ``markuper.pipeline_parser`` can be private, as FOSCommentBundle will never retrieve it dynamically using this name)